sdks/python/container/base_image_requirements.txt

```diff
@@ -35,37 +36,53 @@ oauth2client==4.1.3
 protobuf==3.12.2
 pyarrow==3.0.0
 pydot==1.4.1
-pymongo==3.9.0
-pytz==2019.3
+pymongo==3.10.1
+pytz==2020.1
 pyyaml==5.4
 typing-extensions==3.7.4.3
 
 # GCP extra features
+google-auth==1.31.0
 google-api-core==1.22.0
-google-apitools==0.5.28
+google-apitools==0.5.31
 google-cloud-pubsub==1.0.2
 google-cloud-bigquery==1.26.1
 google-cloud-bigtable==1.0.0
 google-cloud-core==1.4.1
 google-cloud-datastore==1.15.3
+google-cloud-dlp==0.13.0
+google-cloud-language==1.3.0
 google-cloud-profiler==3.0.4
+google-cloud-spanner==1.13.0
+google-cloud-videointelligence==1.13.0
+google-cloud-vision==0.42.0
+google-python-cloud-debugger == 2.15 ; python_version>="3.6"
```

Comment:
we can remove the version guard.

Reply:
Also the pythondocker precommit is failing:
```
17:57:19 RuntimeError: Could not retrieve licences for packages ['bs4', 'google-python-cloud-debugger'] in Python3.6 environment. 
```